app/view_models/workarea/storefront/google_product_feed_sku_view_model.rb

```diff
@@ -17,9 +17,20 @@ def size
         Array.wrap(model.fetch_detail('size')).first
       end
 
+       def inventory
+        inventory_sku? ? inventory_sku.available : 0
```

Comment:
This should be `inventory.available_to_sell` so it respects the logic of the inventory policy. For example backordered may have `0` available, but `10` ready to sell that ship in a week.

Reply:
I've having trouble implementing this. options['inventory_sku'] is an instance of Workarea::Inventory::Collection and the available_to_sell function is not helpful.

Was looking for some guidance here.

Thanks!

Reply:
Hmm, I'm confused on two things.

First, `available_to_sell` is defined on an `Inventory::Collection`, see here: https://github.com/workarea-commerce/workarea/blob/master/core/app/models/workarea/inventory/collection.rb#L33

Second, this looks like it would be an `Inventory::Sku`, not a collection, but maybe I'm reading it wrong:
https://github.com/workarea-commerce/workarea-google-product-feed/blob/master/app/view_models/workarea/storefront/google_product_feed_view_model.rb#L72
https://github.com/workarea-commerce/workarea/blob/master/core/app/models/workarea/inventory/collection.rb#L52
https://github.com/workarea-commerce/workarea/blob/master/core/app/models/workarea/inventory/collection.rb#L75